.github/workflows/node.js.yml

```diff
@@ -0,0 +1,29 @@
+name: Create release tarball and attach to tag
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14"
+      - run: yarn install
+      - run: yarn build
+      - run: |
+          version=`git describe --dirty --tags || echo unknown`
+          mkdir -p dist
+          cp -r build synapse-admin-$version
+          tar chvzf dist/synapse-admin-$version.tar.gz synapse-admin-$version
+          rm -r synapse-admin-$version
+      - uses: softprops/action-gh-release@b7e450da2a4b4cb4bfbae528f788167786cfcedf
```

Comment:
Any reason for using a commit id over a version name? https://github.com/marketplace/actions/gh-release

Reply:
This is as per security recommendation: https://docs.github.com/en/actions/learn-github-actions/security-hardening-for-github-actions#using-third-party-actions

This mitigates the risk that softprops/action-gh-release getting some backdoor added and then published under the same v1.

Reply:
But if this is not important and you trust the owners of softprops/action-gh-release otherwise, I can change it to eg v1.